packages/modules/error-handling-react/package.json

```diff
@@ -56,7 +56,6 @@
     "@bentley/ui-core": "^2.14.0",
     "@svgr/cli": "^5.3.1",
     "@testing-library/react": "^10.4.4",
-    "@types/classnames": "^2.2.10",
```

Comment:
i would bump classnames to v^2.3.1 if youre removing the typedefs, that ver should have the types for the pkg

Reply:
good catch. done